### PR TITLE
[Ahuna] New 'My assistants' page

### LINKED ISSUE #3261: Update My Assistant page
As per [here](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4050-22261&mode=design&t=Dlhq25QyeMahW2OI-4)

Changes including (but check figma for others)

- Opens in a Fullscreen Modal
- Company Assistant can no longer be "de-activated" (no toggle)
- Types of assistants are now separated by Sections (Personal / Shared / Company)
- Small layout changes and use of medium size search bar

Full [figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4050-22251&mode=design&t=Dlhq25QyeMahW2OI-0) (source of truth)

Full [issue](https://github.com/dust-tt/decisions/issues/147#issue-2081773564)

### PR DESCRIPTION
**Pull request by philipperolet**
## Description
Fixes #3261 

image

## Risk
RAS

### CODE REVIEW

File: front/pages/w/[wId]/assistant/assistants.tsx [hunk 1]
```diff
@@ -69,105 +49,47 @@ export const getServerSideProps = withGetServerSidePropsLogging<{
...
-    }
-  };
-
   return (
     <AppLayout
       subscription={subscription}
       owner={owner}
+      hideSidebar
+      titleChildren={
+        <AppLayoutSimpleCloseTitle
```
Comment (flvndvd) [inline front/pages/w/[wId]/assistant/assistants.tsx:91]: Why do we need to use `AppLayoutSimpleCloseTitle`? If I understand correctly, this is useful to have a "save"/"cancel" button in the top bar, which is not the case on this screen, right?

File: front/pages/w/[wId]/assistant/assistants.tsx [hunk 2]
```diff
@@ -183,201 +108,94 @@ export default function PersonalAssistants({
...
-          agentConfiguration={showRemovalModal}
-          show={!!showRemovalModal}
-          onClose={() => setShowRemovalModal(null)}
-          onRemove={() => {
-            void mutateAgentConfigurations();
+      <div className="flex flex-col gap-4 pt-9">
+        <Searchbar
+          name="search"
+          size="md"
+          placeholder="Assistant Name"
```
Comment (flvndvd) [inline front/pages/w/[wId]/assistant/assistants.tsx:115]: Should we be consistent with the gallery screen? If yes, should be "Search (Name)".

File: front/pages/w/[wId]/assistant/assistants.tsx [hunk 3]
```diff
@@ -183,201 +108,94 @@ export default function PersonalAssistants({
...
+        title={SCOPE_INFO[scope].label + "s"}
+        description={SCOPE_INFO[scope].text}
+      />
+      {filteredList.map((agent) => (
+        <ContextItem
+          key={agent.sId}
+          title={`@${agent.name}`}
+          subElement={
+            <span className="text-xs text-element-700">
+              Authors would be here
```
Comment (flvndvd) [inline front/pages/w/[wId]/assistant/assistants.tsx:187]: ??

**Review by flvndvd — COMMENTED** [review]
LGTM left a few comments?

File: front/pages/w/[wId]/assistant/assistants.tsx (hunk 3 above)
Comment (philipperolet) [reply front/pages/w/[wId]/assistant/assistants.tsx:187]: Oh my !

File: front/pages/w/[wId]/assistant/assistants.tsx (hunk 1 above)
Comment (philipperolet) [reply front/pages/w/[wId]/assistant/assistants.tsx:91]: Hmm as I understand it's just to have the close button, which we need there

**Comment by philipperolet** [comment]
@Duncid the sparkle ContextItem subtitle doesn't match the figma for My Assistants contextitems (bigger text, text-600 vs text-700)
image
=> I will update sparkle, to enforce contextitem subtitle consistency across all app. Tell me if instead I should only update this specific use of contextItem in my assistants and leave sparkle as is

**Review by Duncid — APPROVED** [review]

File: front/pages/w/[wId]/assistant/assistants.tsx (hunk 1 above)
Comment (flvndvd) [reply front/pages/w/[wId]/assistant/assistants.tsx:91]: Indeed I missed that it's a "modal".

**Review by flvndvd — APPROVED** [review]
LGTM

**Pull request merged**